server/src/test/java/org/candlepin/policy/AutobindRulesTest.java

```diff
@@ -3078,13 +3092,13 @@ public void testSelectBestPoolsRoleMatchShouldOverrideAllOtherSyspurposeMismatch
         // --- No satisfied syspurpose attributes on the consumer ---
 
         Product prod1 = createSysPurposeProduct(null, "RHEL Workstation", "mismatched_addon",
-            "mismatched_sla", "mismatched_usage");
+            "mismatched_sla", "mismatched_usage", null);
```

Comment:
The consumer should have a service_type set, and this should be "random_service_type" (that mismatches with the consumer's)

Reply:
Done.